support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -543,7 +541,7 @@ export function CheckoutComponent({
 			? 'ANNUAL'
 			: 'ONE_OFF';
 
-	const thresholdAmount = getLowerProductBenefitThreshold(
+	const supporterPlusPrice = getLowerProductBenefitThreshold(
```

Comment:
@paul-daniel-dempsey I was just looking at the `getLowerProductBenefitThreshold` function definition, as I wondered whether `getSupporterPlusPrice` would be a more appropriate name, however I see it takes a `product` parameter, and that could be S+ or Tier 3, so maybe the `supporterPlusPrice` variable isn't appropriate here, as it could in theory be the Tier 3 price? 

It looks like it could be renamed `productPrice`, but I'm not sure how this variable is used, particularly in the event the product is not S+ or Tier 3?

Reply:
I think the issue here is simply coming up with meaningful variable names, I'd be up for looking at this together?